wintermotes/index.html

```diff
@@ -20,5 +33,51 @@
 	<!-- 8. All messages must be printed in the order they where submitted -->
 
 	<div id="message-container"></div>
+
+	<script language="Javascript">
+	var messages = []
+	var container = document.getElementById('message-container')
+
+	function showMessage(){
+		var message = document.getElementById('messageValue')
+		
+		
+
+		if(validateString(message.value)){
+			container.innerHTML = container.innerHTML + "<p>" + message.value + "</p>"
+			messages.push(message.value)
+		} else {
+			console.log("String had number")
+		}
+	}
+
+	function validateString(stringToValidate){
+		// You could also use the .js built-in functions, e.g string.NaN or isNumeric 
```

Comment:
Got it from here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN

But i'm not sure. Haven't tested it. But still, there could be some prebuilt .js functions that might be relevant.